.claude/skills/file-ticket/SKILL.md

```
@@ -0,0 +1,97 @@
+as `-d "$(cat <body-file>)"`.
+
+**GitHub.** No templates to check for railyard's own repos. `gh` does take a
+body file, `-F`. A pull request is `#N` in issue text, so the autolink works.
+
+## 6. File it
+
+Skim the title and opening paragraph the way a human would — that skim is only
+possible before the post. Check the one CLI you are about to use is present,
+not both:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)